### PR TITLE
added support for FosRestBundle 2.0

### PR DESCRIPTION
**Pull request by lemorragia**
## Subject
This is my first ever contribution on github, and also my first pull request, so go easy on me ;-). 
Anyway, following the support for FosRestBundle 2.0 in SonataUserBundle, i made this pull request to add support for FosRestBundle >= 2.0 to the 3.x branch.

Critiques are welcome, especially if i did something wrong.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
         "symfony/options-resolver": "^2.3 || ^3.0"
     },
     "require-dev": {
-        "friendsofsymfony/rest-bundle": "^1.1",
+        "friendsofsymfony/rest-bundle": "<1.1",
```
Comment (core23) [inline composer.json]: This looks wong. Why should we load a lower version?

Comment (lemorragia) [inline composer.json]: it is wrong. I confused the confict and require-dev section. Already made a fix...if i'm not wrong it should be automatically added here, right?

Comment (greg0ire) [inline composer.json]: Yes it was added… would have been better to amend your commit and force push it, but it worked.

**Comment by greg0ire** [comment]
> Anyway, following the support for FosRestBundle 2.0 in SonataUserBundle

@lemorragia is reffering to https://github.com/sonata-project/SonataUserBundle/pull/802

**Comment by OskarStark** [comment]
Thank you @lemorragia 👍

**Comment by Soullivaneuh** [comment]
Please next time ask for a changelog before merging.

**Pull request merged**